### ISSUE TITLE
Users breadcrumb in Admin wrong link

### ISSUE THREAD

**Issue by simplybenuk** [nhsengland/futurenhs#324]
As a platform administrator:

1. Go to admin
2. Go to manage users
3. Click on any user to perform an action
4. Clicking on users breadcrumb should take admin back to the manage users page but it takes user to my groups page
![image](https://user-images.githubusercontent.com/9471595/179695344-97161b98-3393-45c8-a1d6-07206ee5c0d1.png)

**Comment by ReallySmall** [comment]
We potentially need a story for this. The issue is generic and this is just an example of it.

/users is a necessary route to encapsulate all user routes, but it doesn't currently render a view itself. The current behaviour with routes like this is to redirect to "/" when accessed. As we're currently hiding the home route and redirecting to /groups, that's why you see the current behaviour.

I suggest that the breadcrumb needs to take an additional property for each route segment which allows us to render specific breadcrumbs as plain text rather than links

**Comment by ReallySmall** [comment]
I have a feeling that @Alex-Grubba-CDS logged an issue for this

**Comment by Alex-Grubba-CDS** [comment]
Yes I believe we have a story for Breadcrumb behaviour that in part is covering this issue as I raised it a while back.
